### PR TITLE
git-branchless 0.7.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Porkepix** [comment]
Licence changed to MIT & Apache 2.0 (https://github.com/arxanas/git-branchless/commit/fe2c999b60c19bbdcba202fd49322a8ebdc7434d)
Also, this release isn't tagged yet, see https://github.com/arxanas/git-branchless/issues/842 which was for test purpose and should end up with a GitHub release soon.

**Comment by chenrui333** [comment]
relates to https://github.com/arxanas/git-branchless/discussions/813

**Comment by arxanas** [comment]
> Licence changed to MIT & Apache 2.0 (https://github.com/arxanas/git-branchless/commit/fe2c999b60c19bbdcba202fd49322a8ebdc7434d)

Although the license is changed in the latest `master`, the licensing change is not actually included in v0.7.1, since I considered that a breaking change. It will be in v0.8.0. You can see the full changelog here: https://github.com/arxanas/git-branchless/compare/v0.7.0...v0.7.1

**Comment by Porkepix** [comment]
> > Licence changed to MIT & Apache 2.0 ([arxanas/git-branchless@fe2c999](https://github.com/arxanas/git-branchless/commit/fe2c999b60c19bbdcba202fd49322a8ebdc7434d))
> 
> Although the license is changed in the latest `master`, the licensing change is not actually included in v0.7.1, since I considered that a breaking change. It will be in v0.8.0. You can see the full changelog here: [arxanas/git-branchless@v0.7.0...v0.7.1](https://github.com/arxanas/git-branchless/compare/v0.7.0...v0.7.1)

Mmmmh. Dunno how the Formula audit will manage it then, because here it was complaining of the old licence being wrong compared to what was seen when checking GitHub's repo.

Why would a change to a more permissive licence be a breaking one, though? I'd understand if the change was toward a more restrictive one, as people allowed to use it might not be allowed anymore, but here it'd only allow more people to use it while not forcing any new constraints on current users.

**Comment by chenrui333** [comment]
the license audit is using github license API, which is whatever in the main branch. (but it can be safely ignored as audit failure though)

**Comment by chenrui333** [comment]
@arxanas 👋 question for you, is 0.7.1 the latest release? If so, can you mark it as the latest release in the github side? Thanks!

**Comment by chenrui333** [comment]
https://github.com/arxanas/git-branchless/issues/854

**Comment by arxanas** [comment]
@chenrui333 Released https://github.com/arxanas/git-branchless/releases/tag/v0.7.1, thanks for packaging!

You or someone else may be interested in using the `install-man-pages` subcommand in v0.7.1 to install man-pages as part of installation.

**Comment by chenrui333** [comment]
Thanks @arxanas!

**Comment by chenrui333** [comment]
Only the audit failure for license mismatch, good to be merged.

**Comment by p-linnane** [comment]
Can we add it to https://github.com/Homebrew/homebrew-core/blob/master/audit_exceptions/permitted_formula_license_mismatches.json?

File: Formula/git-branchless.rb [hunk 1]
```diff
@@ -25,6 +25,10 @@ def install
     system "cargo", "install", *std_cargo_args(path: "git-branchless")
   end
 
+  def post_install
+    system "git", "branchless", "install-man-pages", man1
+  end
```
Comment (carlocab) [inline Formula/git-branchless.rb:30]: Can we do this in `install`?

Comment (chenrui333) [reply Formula/git-branchless.rb:30]: we can't, git actually complains that it cannot find `branchless` command.

**Comment by chenrui333** [comment]
> Can we add it to https://github.com/Homebrew/homebrew-core/blob/master/audit_exceptions/permitted_formula_license_mismatches.json?

I left a note about the license thing, it can be updated in the next release 0.8.x

File: Formula/git-branchless.rb (hunk 1 above)
Comment (carlocab) [reply Formula/git-branchless.rb:30]: Have you tried doing
```ruby
ENV.prepend_path "PATH", bin
```
?

Comment (chenrui333) [reply Formula/git-branchless.rb]: that is a good point 👍

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**